src/jit/emitxarch.cpp

```diff
@@ -363,6 +363,34 @@ bool emitter::IsDstSrcSrcAVXInstruction(instruction ins)
     }
 }
 
+//------------------------------------------------------------------------
+// IsDstSrcImmAvxInstruction: check if instruction has "R(M) R(M) I" format
+// for EVEX, VEX and legacy SSE encodings and has no (E)VEX.NDS
+//
+// Arguments:
+//    instruction -- processor instruction to check
+//
+// Return Value:
+//    true if instruction has "R(M) R(M) I" format and has no (E)VEX.NDS
```

Comment:
It might be helpful to elucidate just a bit more what this actually means (I think that it means that it doesn't have two register sources, so we don't have to worry about which one is overwritten)

Reply:
I see that this was actually just moved from below - still a bit more explanation would be useful :-)

Reply:
Fixed.